devlab/provision/generate_load.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+set -e
+# set -x
+
+export OS_TENANT_NAME=admin
+export OS_USERNAME=admin
+export OS_PASSWORD=admin
+export SERVICE_TOKEN=ADMIN
+export OS_AUTH_URL="http://192.168.1.2:5000/v2.0/"
```

Comment:
Looks like this script is for grizzly only. So, it should be named appropriately or re-written to work for other versions. In second case IPs can be retrieved from the configuration file.